`make build` not working after multiple arch makefile update

## Bug Report

`make build` results in the following error:
```
BUILD_VERSION: v0.8.0-dev.1
CONFIG_VERSION: v0.8.0
FRAMEWORK_BUILD_VERSION: v1.4.0-pre-alpha-2
XDG_DATA_HOME: /home/runner/.local/share
rm -rf ./artifacts
make[1]: Entering directory '/home/runner/work/community-edition/community-edition/hack/builder'
TANZU_CLI_NO_INIT=true GOPRIVATE="github.com/vmware-tanzu/*,github.com/dvonthenen/*,github.com/joshrosso/*" go run -ldflags "-s -w -X "github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli.BuildDate=$(date -u +"%Y-%m-%d")" -X "github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli.BuildSHA=$(git rev-parse --short HEAD)" -X "github.com/vmware-tanzu/tanzu-framework/pkg/v1/cli.BuildVersion=$(git describe --tags --abbrev=0)" -X 'main.BuildEdition=tce-standalone' -X 'github.com/vmware-tanzu/tanzu-framework/pkg/v1/tkg/buildinfo.IsOfficialBuild=""'" github.com/vmware-tanzu/tanzu-framework/cmd/cli/tanzu \
	plugin install all --local ../.././artifacts
Error: could not find manifest.yaml file: open ../../artifacts/manifest.yaml: no such file or directory

✖  could not find manifest.yaml file: open ../../artifacts/manifest.yaml: no such file or directory 
exit status 1
make[1]: *** [Makefile:74: install-plugins] Error 1
make[1]: Leaving directory '/home/runner/work/community-edition/community-edition/hack/builder'
make: *** [Makefile:245: install-cli-plugins] Error 2
Error: Process completed with exit code 2.
```

## Expected Behavior

`make build` should still work after the multiple-architecture makefile rework which dropped the `build-cli-plugins` step before isntall: https://github.com/vmware-tanzu/community-edition/pull/1158/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52L225

## Steps to Reproduce the Bug

`make build` locally ([and in CI](https://github.com/vmware-tanzu/community-edition/runs/3477814062))

## Environment Details

N/a

--- a/Makefile
+++ b/Makefile
@@ -241,7 +241,7 @@ build-cli-plugins-%: prep-build-cli
 build-cli-plugins-local: build-cli-plugins-${GOHOSTOS}-${GOHOSTARCH}
 
 .PHONY: install-cli-plugins
-install-cli-plugins:
+install-cli-plugins: build-cli-plugins
 	@cd ./hack/builder/ && $(MAKE) install-plugins
 
 .PHONY: build-install-plugins

